CMakeLists.txt

```diff
@@ -56,6 +56,7 @@ if(CROSS_COMPILE)
 	include(cmake/RaspberryPi.cmake)
 	# Raspbian ships CLang 3.9 in its repositories
 	set(CLANG_FOUND /usr/bin/clang-3.9)
+        set(OPT_FOUND /usr/bin/opt-3.9)
```

Comment:
Is opt-3.9 automatically installed with CLang-3.9?

Reply:
No, `llvm3.9` include `opt-3.9`. 
Currently vc4c itself depends on `llvm-3.9`. so this wont be a problem.